components/steering_wheel/src/IO_Digital.c

```diff
@@ -0,0 +1,64 @@
+/*
+ * IO_Digital.c
+ * Digital IO Module source code
+ */
+
+/******************************************************************************
+ *                             I N C L U D E S
+ ******************************************************************************/
+
+// Module Header
+#include "IO_Digital.h"
+
+// System includes
+#include <string.h>
+
+// Other includes
+#include "ModuleDesc.h"
+#include "Utility.h"
+#include "SystemConfig.h"
+
+/******************************************************************************
+ *                           P U B L I C  V A R S
+ ******************************************************************************/
+
+IO_Digital_S IO_DIGITAL;
+
+/******************************************************************************
+ *                       P U B L I C  F U N C T I O N S
+ ******************************************************************************/
+
+/*
+ * @brief Initializes the Digital IO Init
+ */
+static void IO_Digital_init(void)
+{
+    memset(&io_digital, 0, sizeof(IO_Digital_S));
```

Comment:
second parameter should be `0x00` or `0U` (I prefer `0x00`) for clarity and to make sure the compiler does what it should